packages/config/src/layer2s/mode.ts

```diff
@@ -65,14 +65,47 @@ export const mode: Layer2 = opStackL2({
   },
   nonTemplatePermissions: [
     ...discovery.getMultisigPermission(
-      'ModeMultisig',
-      'This address is the owner of the following contracts: ProxyAdmin, SystemConfig. It is also designated as a Guardian of the OptimismPortal, meaning it can halt withdrawals. It can upgrade the bridge implementation potentially gaining access to all funds, and change the sequencer, state root proposer or any other system component (unlimited upgrade power).',
+      'ProxyAdminOwner',
+      'Owner of the ProxyAdmin: it can upgrade the bridge implementation potentially gaining access to all funds, and change any system component. Also designated as the owner of the SystemConfig, meaning it can update the preconfer address, the batch submitter address and the gas configuration of the system.',
+    ),
+    discovery.contractAsPermissioned(
+      discovery.getContract('SuperchainProxyAdmin'),
+      'Admin of the shared SuperchainConfig contract.',
+    ),
+    ...discovery.getMultisigPermission(
+      'SuperchainProxyAdminOwner',
+      'Owner of the SuperchainProxyAdmin.',
+    ),
+    ...discovery.getMultisigPermission(
+      'FoundationMultisig_1',
+      'Member of the ProxyAdminOwner.',
+    ),
+    ...discovery.getMultisigPermission(
+      'SecurityCouncilMultisig',
+      'Member of the ProxyAdminOwner.',
+      [
+        {
+          text: 'Security Council members - Optimism Collective forum',
+          href: 'https://gov.optimism.io/t/security-council-vote-2-initial-member-ratification/7118',
+        },
+      ],
+    ),
+    ...discovery.getMultisigPermission(
+      'FoundationMultisig_2',
+      'This address is designated as a Guardian of the OptimismPortal, meaning it can halt withdrawals.',
     ),
     ...discovery.getMultisigPermission(
       'ChallengerMultisig',
       'This address is the permissioned challenger of the system. It can delete non finalized roots without going through the fault proof process. It is also designated as the Guardian.',
     ),
   ],
+  nonTemplateContracts: [
+    discovery.getContractDetails('SuperchainConfig', {
```

Comment:
this should have `SuperchainProxyAdmin` as upgradeability but currently has the proxyadmin

Reply:
haha it seems i can't quote but i do mean the SuperchainConfig's upgradeability